Cargo.toml

```diff
@@ -34,11 +36,9 @@ doctest = false
 debugmozjs = ['mozjs_sys/debugmozjs']
 promises = ['mozjs_sys/promises']
 
-[dependencies.mozjs_sys]
-git = "https://github.com/servo/mozjs"
-
 [dependencies]
 lazy_static = "0.2.1"
 libc = "0.2"
 log = "0.3"
 num-traits = "0.1.32"
+mozjs_sys = "0.0.0"
```

Comment:
The mozjs_sys version on crates.io is 0.1.0 instead of 0.0.0. https://crates.io/crates/mozjs_sys

Reply:
Huh. I didn't realize it had ever been published before.

Reply:
@SimonSapin, can you add myself and others (jdm, nox, asajeffrey, KiChjang, etc?) as owners? Thanks!

Reply:
@fitzgen I’ve added you. The `servo/cargo-publish` team already has access.

Reply:
Since we’re moving things around anyway we could pick one of `mozjs` vs `js` vs something else for both crates?

Reply:
D'oh -- I meant to rename this crate to `mozjs` since `js` is already taken on crates.io. Good eye!